.github/workflows/kitchen.vagrant.yml

```diff
@@ -41,8 +41,8 @@ jobs:
       fail-fast: false
       matrix:
         instance:
-          - py3-git-3000-freebsd-130
-          - py3-git-3000-freebsd-122
+          # - py3-git-3000-freebsd-130
+          # - py3-git-3000-freebsd-122
```

Comment:
Before submitting this PR, I ran the tests in my fork and these two jobs were failing, so I disabled them for the time being:

* https://github.com/myii/salt-bootstrap/actions/runs/1029627093

Perhaps someone will be able to get them working again eventually.  @krionbsd Any thoughts?

Reply:
We have to exclude 3000 from CI anyway, FreeBSD merged Python 3.8 to 12.2 as default python version, and I failed to use it with salt 3000